### PR TITLE
Gutenboarding Design Selector: Add heading, srcset

### PR DESCRIPTION
**Pull request by ockham**
#### Changes proposed in this Pull Request

Based on #38456. Add headings (according to mockup) and `srcset` for moar high-res images.

#### Testing instructions

http://calypso.localhost:3000/gutenboarding

### CODE REVIEW

**Comment by ockham** [comment]
Rebased.

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+	}
+
+	h2,
+	h3 {
+		color: #23282d;
+		font-size: 1.3em;
+		margin-bottom: 3em;
+		text-align: center;
+	}
+}
```
Comment (sirreal) [inline client/landing/gutenboarding/onboarding-block/design-selector/style.scss]: I'd prefer adding a specific classname for styling. It seems like using generic elements (even scoped like this) tends to come back to bite you.

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+		color: #23282d;
+		font-size: 2em;
+		font-weight: bold;
+		margin-bottom: 0.25em;
+		text-align: center;
+	}
+
+	h2,
+	h3 {
+		color: #23282d;
```
Comment (simison) [inline client/landing/gutenboarding/onboarding-block/design-selector/style.scss]: Are there any sass variables available for these?

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss (hunk 1 above)
Comment (ockham) [reply client/landing/gutenboarding/onboarding-block/design-selector/style.scss]: Okay, can do

**Review by sirreal — APPROVED** [review]
Looks great

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss (hunk 2 above)
Comment (ockham) [reply client/landing/gutenboarding/onboarding-block/design-selector/style.scss]: Yep: 0b4134de7d

**Pull request merged**